src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs

```diff
@@ -164,7 +164,7 @@ public void GetExecutablePath_should_find_existing_executable()
             [Fact]
             public void GetExecutablePath_should_find_existing_executable_with_extension()
             {
-                FileSystem.get_executable_path("cmd.exe").ShouldEqual(
+                FileSystem.get_executable_path("cmd.exe").ToLower().ShouldEqual(

```

Comment:
It's more weird in that you are shifting it to lower and asking if it is equal to a string with upppercase.